zerver/lib/digest.py

```diff
@@ -255,18 +255,16 @@ def gather_new_streams(
     else:
         new_streams = [stream for stream in recently_created_streams if stream.is_web_public]
 
-    base_url = f"{realm.uri}/#narrow/stream/"
-
-    streams_html = []
-    streams_plain = []
+    channels_html = []
+    channels_plain = []
 
     for stream in new_streams:
-        narrow_url = base_url + encode_stream(stream.id, stream.name)
```

Comment:
This was basically doing the same thing as stream_narrow_url and updating it now will makes the URL update commits cleaner.

Reply:
Cool, yeah this code path is very stale.